.github/workflows/tests.yml

```diff
@@ -15,15 +15,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        PYTHON_VERSION: ['3.7', '3.9', '3.10']
+        os: [ubuntu, windows, macos]
+        PYTHON_VERSION: ['3.8', '3.12']
         PIP_SELECTOR: ['[tests]']
```

Comment:
The `PIP_SELECTOR` is missing for the `3.8` and `3.12` build, maybe there is some parsing issue and it needs a coma to make sure it is a list?